egui/src/widgets/plot/mod.rs

```diff
@@ -67,11 +67,34 @@ impl Default for CoordinatesFormatter {
 
 const MIN_LINE_SPACING_IN_POINTS: f64 = 6.0; // TODO: large enough for a wide label
 
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(Clone)]
+pub struct AutoBounds {
```

Comment:
```suggestion
struct AutoBounds {
```

I don't think it needs to be `pub`?